code/modules/admin/verbs/randomverbs.dm

```diff
@@ -89,6 +89,7 @@
 
 	log_admin("[key_name(usr)]: Modified [key_name(C)]'s antagonist reputation [log_text]")
 	message_admins("<span class='adminnotice'>[key_name_admin(usr)]: Modified [key_name(C)]'s antagonist reputation ([log_text])</span>")
+	to_chat(C, "<span class='userdanger'>[key_name_admin(usr)]: Modified your antagonist reputation ([log_text])</span>")

```

Comment:
`key_name_admin` will include a PM link and the admin character's name, do you want that?

Reply:
Should probably just be usr.key

Reply:
Not that it's a bad thing but this will reveal stealthmins.